### PR TITLE
Duplicate projects and build version specific bundles

### LINKED ISSUE #4753: MP FT 1.1 changes leaks to MP FT1.0
I discovered there is unintended changes required by MP FT 1.1, made to MP FT1.0, which breaks our zero migration principle. We need to fix this before the 18.0.0.3 release.

### PR DESCRIPTION
**Pull request by neilgsyoung**
fixes #4753 

**### L2 and ID please note**: This is a tactical fix in order to make the Q3 deadline. The approach is to duplicate the CDI project, thus creating a FaultTolerance CDI package for 1.0 and for CDI 1.1. Although the code differs between the 2 projects, the messages do not - they are unchanged between FT 1.0 and FT 1.1.

### CODE REVIEW

**Comment by neilgsyoung** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mpFaultTolerance1.1-cdi1.2.feature [hunk 1]
```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.mpFaultTolerance1.0-cdi1.2
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mpFaultTolerance1.1-cdi1.2.feature]: wrong symbolicName

File: dev/com.ibm.ws.microprofile.faulttolerance.1.1.cdi/bnd.bnd [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+WS-TraceGroup: FAULTTOLERANCE
+
+-dsannotations-inherit: true
+-dsannotations: com.ibm.ws.microprofile.faulttolerance.cdi.FaultToleranceCDIExtension,\
+                com.ibm.ws.microprofile.faulttolerance.cdi.MetricComponent
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+-buildpath: \
+	com.ibm.ws.transaction.cdi;version=latest,\
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.1.1.cdi/bnd.bnd]: do you know why this transactions dependency is needed?

File: dev/com.ibm.ws.microprofile.faulttolerance.1.1.cdi/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	com.ibm.ws.logging;version=latest,\
+	com.ibm.websphere.javaee.cdi.1.2;version=latest,\
+	com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
+	com.ibm.ws.cdi.interfaces;version=latest,\
+	com.ibm.websphere.org.eclipse.microprofile.faulttolerance.1.0;version=latest,\
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.1.1.cdi/bnd.bnd]: shouldn't this say 1.1?

File: dev/com.ibm.ws.microprofile.faulttolerance.1.1.cdi/bnd.bnd [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	com.ibm.websphere.org.eclipse.microprofile.config.1.1;version=latest,\
```
Comment (tevans78) [inline dev/com.ibm.ws.microprofile.faulttolerance.1.1.cdi/bnd.bnd]: this should say 1.3 ... actually i have done this already, merge needed

**Comment by neilgsyoung** [comment]
#build

**Comment by lzdanski** [comment]
@neilgsyoung Thanks for the note! If something similar happens in the future, you can use the "ID review not needed" label so we know that the changed files in the diff have been reviewed or do not need any edits.

**Comment by neilgsyoung** [comment]
#build

**Pull request merged**